src/queries/positions.tsx

```diff
@@ -53,6 +53,17 @@ export const GetMultiPositionsQuery = gql`
   ${postitionFragment}
 `
 
+export const PositionsSearchQuery = gql`
+  query PositionsSearch($positionId: String!) {
+    positions(first: 1000, where: { id: $positionId }) {
```

Comment:
Actually, as TheGraph returns at most 1000 elements, the best way would be to have a positions counter in the subgraph and to use that counter to create multiples queries to get the complete set of positions.

Reply:
Pagination pattern (like the use of PAGE_SIZE and OFFSET) besides the orderBy can be configured with some fields by default in any constant. The user should could configure that parameters on the frontend page as well and call the backend with parametrized values. This can be another feature to do in the future for all lists.

Reply:
We can work this improvement in this issue https://github.com/gnosis/conditional-tokens-explorer/issues/125